app/models/checkout.rb

```diff
@@ -101,7 +102,10 @@ def days_remaining
   end
 
   def library
-    fields['library']['key']
+    code = fields['library']['key']
+    return BORROW_DIRECT_CODE if code == 'SUL'
```

Comment:
This only applies to checkouts.  When an ON_ORDER item is requested its library is SUL so we don't want this translation to be universal.